workers-site/index.js

```diff
@@ -1,15 +1,31 @@
-import { getAssetFromKV } from '@cloudflare/kv-asset-handler'
+import { getAssetFromKV, defaultKeyModifier} from '@cloudflare/kv-asset-handler'
+
+// do not set to true in production!
+const DEBUG = false
 
 addEventListener('fetch', event => {
   event.respondWith(handleEvent(event))
 })
 
 async function handleEvent(event) {
+  const url = new URL(event.request.url)
   try {
-    return await getAssetFromKV(event)
+    let options = {}
+    if (DEBUG) {
+      options = {
+        cacheControl: {
+          bypassCache: true
+        }
+      }
+    }
+    return await getAssetFromKV(event, options)
   } catch (e) {
-    return new Response(e.message || e.toString(), {
-      status: 404,
-    })
+    if (DEBUG) {
+      return new Response(e.message || e.toString(), {
+        status: 404,
```

Comment:
```suggestion
        status: 500,
```

Reply:
I think this should still be 404, any errors thrown from the handler mean the asset could not be found